class NotFoundError(Exception):
    """Объект не найден"""

    pass


class AlreadyExistError(Exception):
    """Такой объект уже существует"""

    pass
